flecsi/utils/mpi_type_traits.h

```diff
@@ -121,7 +121,7 @@ struct mpi_typetraits_u {
       return MPI_CXX_DOUBLE_COMPLEX;
     else if constexpr(is_same_v<TYPE, complex<long double>>)
       return MPI_CXX_LONG_DOUBLE_COMPLEX;
-    else if constexpr(is_same_v<TYPE, byte>)
+    else if constexpr(is_same_v<TYPE, std::byte>)
```

Comment:
Why is this necessary if `is_same_v` can be used unqualified?  Is there some other `byte` that makes the lookup ambiguous?

Reply:
This might be a MSVC problem, I'm not sure. I was seeing errors saying that `is_same_v<TYPE, byte>` was ambiguous.

Reply:
It's not as egregious as the `min`/`max` business, but [yes it's Windows](https://stackoverflow.com/questions/58059048/ambiguous-byte-definition-in-rpcndr-and-cstddef).  It wouldn't be a problem if Windows used namespaces, although, to be fair, they did start using C++ in earnest before those existed!

Reply:
Can we leave it as is or would you like me to apply changes?

Reply:
I don't think there's any cleaner way to make it work than what you did.  You could add a comment, I guess, identifying the other `byte` that was ambiguous, so that no one "fixes" it later.